### DISCUSSION TITLE
I have Graphviz installed but -D does not do anything

### DISCUSSION THREAD

**Discussion by kylegoetz** [tree-sitter/tree-sitter#3845] [Q&A] [answered]
### Problem

`tree-sitter parse myfile -D` has the exact same output as without `-D`. Nothing involving Graphviz is done.

However, `which dot` shows that it's installed (via Homebrew).

### Steps to reproduce

come sit at my computer and execute the above command?

### Expected behavior

I expect a visualization to be generated

### Tree-sitter version (tree-sitter --version)

0.24.3

### Operating system/version

macOS 15.0.1

**Comment by clason** [comment, marked as answer]
`tree-sitter parse --help`

**Comment by kylegoetz** [comment]
@clason  awesome, thanks. Last time I did this was on a different machine and -D opened up my browser with the output. I didn't realize the CLI commands changed again.
